packages/framework-core/src/booster-register-handler.ts

```diff
@@ -31,9 +32,16 @@ export class RegisterHandler {
       return
     }
 
-    await config.provider.events.store(
-      register.eventList.map((event) => RegisterHandler.wrapEvent(config, event, register)),
-      config
+    const logger = getLogger(config, 'RegisterHandler#handle')
+
+    await retryWithBackoff(
+      () =>
+        config.provider.events.store(
+          register.eventList.map((event) => RegisterHandler.wrapEvent(config, event, register)),
+          config
+        ),
+      config.eventStoreRetry,
+      logger
     )
```

Comment:
Is correct to assume that if the `store` method throws an error the event is not stored and it is safe to retry, guaranteeing that two copies of the same event are not stored, just only one?